.github/workflows/test.yml

```diff
@@ -2,9 +2,12 @@ name: Tests & Code Coverage
 
 on:
   pull_request:
-  push:
     branches:
       - "**"
+  push:
```

Comment:
So if I updated a PR with a new change, CI wouldn't run again?

Reply:
It would! thats what the pull_request section does

Reply:
https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request

> For example, if no activity types are specified, the workflow runs when a pull request is opened or reopened or when the head branch of the pull request is updated.